package-testing/php-sdk-relay/src/AssignmentHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Eppo\SDKTest;
+
+use Eppo\EppoClient;
+use Eppo\Exception\EppoClientException;
+use Exception;
+
+class AssignmentHandler
+{
+    private static array $methods = [
+        'INTEGER' => 'getIntegerAssignment',
+        'STRING' => 'getStringAssignment',
+        'BOOLEAN' => 'getBooleanAssignment',
+        'NUMERIC' => 'getNumericAssignment',
+        'JSON' => 'getJSONAssignment',
+    ];
+
+    public function __construct(private EppoClient $eppoClient, private TestLogger $assignmentLogger)
+    {
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function getAssignment(array $payload): array
+    {
+        $variationType = $payload['variationType'];
+        $flagKey = $payload['flag'];
+        $default = $payload['defaultValue'];
+        $subjectKey = $payload['subjectKey'];
+        $subjectAttributes = $payload['subjectAttributes'];
+
+        try {
+            if (isset(self::$methods[$variationType])) {
```

Comment:
consider making this a guard clause to spatially locate the error being thrown with why
```php
if (!isset(self::$methods[$variationType])) {
    throw new Exception("Invalid variation type $variationType");   
}

...
```

Reply:
💪